compiler/src/options/mod.rs

```diff
@@ -0,0 +1,121 @@
+use std::collections::HashMap;
```

Comment:
this file is a perfect candidate for adding unit tests

Reply:
Thanks, add task to issue #12